stellar-common/src/payments.rs

```diff
@@ -26,17 +26,37 @@ fn map_payments(block: Block) -> Result<Payments, substreams::errors::Error> {
             Err(_) => return,
         };
 
+        // You may want to consider the fees, which appear at the transaction
+        // level, as a payment, since there will be a debit from the payer's
+        // account, though there is no destination (fees are deposited into an
+        // inaccessible fee pool).
+
         trx.operations.iter().for_each(|operation| match &operation.body {
             stellar_xdr::curr::OperationBody::Payment(payment) => {
+                // It's quite risky to do floating-point division by 1e7 here.
+                // If the amount is small it's likely to result in significant
+                // imprecision errors. In Horizon we have an `amount` helper
+                // package [1] to do fixed point calculations to ensure
+                // correctness, I suggest a `big.Rat` [2] Rust equivalent here.
```

Comment:
Yeah, we can use big decimal or something like that and change the output from an f64 to a string